src/neptune/metadata_containers/run.py

```diff
@@ -234,6 +236,8 @@ def __init__(
                 Defaults to True.
                 The tracked metadata is stored in the '<monitoring_namespace>/traceback' namespace (see the
                 'monitoring_namespace' parameter).
+            git_ref: `GitRef` object containing information about the git repository path.
```

Comment:
@normandy7 that's something for you. We should probably think of mentioning how to disable repository tracking with `GitRef.DISABLED`.